src/main.py

```diff
@@ -284,14 +284,25 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
         else:
```

Comment:
up above, right after the line "for test_task_idx, task in enumerate(test_tasks):" can you add the following comment:

Run the approach's solve() method to get a policy for this task.